orchestrator/assets/business_travel.py

```diff
@@ -194,3 +202,70 @@ def mode_co2_mapper(s3: S3Resource):
     mapper = pd.read_csv(obj["Body"])
     mapper = mapper.rename(columns={"Transport Mode": "transport_mode", "CO2 Factor": "CO2_factor"})
     return mapper
+
```

Comment:
3 assets: all_scope_summary loaded from Datahub, cost_object_warehouse, loaded from MIT warehouse, and the last sync a Postgres table to Datahub.